spec/presenters/result_set_presenter_spec.rb

```diff
@@ -97,6 +105,21 @@
       presenter.stub(:describe_filters_in_sentence).and_return("a sentence summarising the selected filters")
       presenter.stub(:documents).and_return({ key: 'value' })
       presenter.stub(:any_filters_applied?).and_return(true)
+      view_context.stub(:render) { '<nav class="govuk-previous-and-next-navigation" role="navigation" aria-label="Pagination">
+        <ul class="group">
+          <li class="previous-page">
+            <a href="previous-page" rel="previous">
+              <span class="pagination-part-title">Previous page</span>
+              <span class="pagination-label">1 of 2</span>
+            </a>
+          </li>
+          <li class="next-page">
+            <a href="next-page" rel="next">
+              <span class="pagination-part-title">Next page</span>
+              <span class="pagination-label">2 of 2</span>
+            </a>
+          </li>
+      </ul></nav>' }
```

Comment:
I'm not sure if actually having this markup here affects the spec? If not, it might be clearer to just return a smaller string, like `"<nav></nav>"`?